client/src/protocol.ts

```diff
@@ -556,6 +557,10 @@ export interface ServerCapabilities {
 	 */
 	referencesProvider?: boolean;
 	/**
+	 * The server provides workspace references exporting support.
+	 */
+	xworkspaceReferencesProvider?: boolean;
```

Comment:
This is the model for the json response so it needs to match the spec exactly:
https://github.com/sourcegraph/language-server-protocol/blob/master/extension-workspace-references.md